examples/complete-mssql/main.tf

```diff
@@ -1,33 +1,64 @@
 provider "aws" {
-  region = "us-east-1"
+  region = local.region
 }
 
 locals {
+  name   = "complete-mssql"
+  region = "eu-west-1"
   tags = {
     Owner       = "user"
     Environment = "dev"
   }
 }
 
-##############################################################
-# Data sources to get VPC, subnets and security group details
-##############################################################
-data "aws_vpc" "default" {
-  default = true
-}
+################################################################################
+# Supporting Resources
+################################################################################
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.77"
```

Comment:
I think that we can set `version` to be `~> 2`, so that we _also_ test compatibility of the whole range of versions and not just specific version 2.77. What do you think?

Reply:
sounds good - updated!